src/MCPClient-base.Dockerfile

```diff
@@ -82,15 +82,15 @@ RUN set -ex \
 
 # OS dependencies from .deb files
 RUN set -ex \
-  && curl -s https://mediaarea.net/download/binary/libzen0/0.4.34/libzen0_0.4.34-1_amd64.xUbuntu_14.04.deb --output libzen0_0.4.34-1_amd64.xUbuntu_14.04.deb \
-  && curl -s https://mediaarea.net/download/binary/libmediainfo0/0.7.91/libmediainfo0_0.7.91-1_amd64.xUbuntu_14.04.deb --output libmediainfo0_0.7.91-1_amd64.xUbuntu_14.04.deb \
-  && curl -s https://mediaarea.net/download/binary/mediaconch/16.12/mediaconch_16.12-1_amd64.xUbuntu_14.04.deb --output mediaconch_16.12-1_amd64.xUbuntu_14.04.deb \
-  && dpkg -i libzen0_0.4.34-1_amd64.xUbuntu_14.04.deb \
-  && dpkg -i libmediainfo0_0.7.91-1_amd64.xUbuntu_14.04.deb \
-  && dpkg -i mediaconch_16.12-1_amd64.xUbuntu_14.04.deb \
-  && rm libzen0_0.4.34-1_amd64.xUbuntu_14.04.deb \
-  && rm libmediainfo0_0.7.91-1_amd64.xUbuntu_14.04.deb \
-  && rm mediaconch_16.12-1_amd64.xUbuntu_14.04.deb
+  && curl -s https://mediaarea.net/download/binary/libzen0/0.4.37/libzen0_0.4.37-1_amd64.xUbuntu_14.04.deb --output libzen0_0.4.37-1_amd64.xUbuntu_14.04.deb \
+  && curl -s https://mediaarea.net/download/binary/libmediainfo0/17.12/libmediainfo0_17.12-1_amd64.xUbuntu_14.04.deb --output libmediainfo0_17.12-1_amd64.xUbuntu_14.04.deb \
+  && curl -s https://mediaarea.net/download/binary/mediaconch/17.12/mediaconch_17.12-1_amd64.xUbuntu_14.04.deb --output mediaconch_17.12-1_amd64.xUbuntu_14.04.deb \
```

Comment:
Do we want to bring dependencies such as this onto the AM package server? `http://packages.archivematica.org/1.7.x/ubuntu-externals trusty main"` I think JHOVE and pals are there, but I'm not sure what the process/policy looks like on this.

Reply:
@ross-spencer I think there has been some talk about this in the past. @scollazo do you have an opinion on this?

Reply:
This came up in convo with MediaArea team when we were establishing the integration. MediaArea team were happy to host the binaries -- especially since things were more heavily in movement, but the original Artefactual team proposal involved pulling into the AM package server. IIRC, when I was only on MediaArea team instead of now, when I am here on both teams.

Reply:
The convo I have about this is from December 2016 (!) with @scollazo saying he prefers adding a mediaconch repo to the install procedures and Jerome/MediaArea saying only the .deb files were currently available. It's likely that has changed.